test/Core.Test/AdminConsole/OrganizationFeatures/Policies/PolicyRequirements/MasterPasswordPolicyRequirementTests.cs

```diff
@@ -11,6 +11,21 @@ namespace Bit.Core.Test.AdminConsole.OrganizationFeatures.Policies.PolicyRequire
 [SutProviderCustomize]
 public class MasterPasswordPolicyRequirementFactoryTests
 {
+    [Fact]
+    public void MasterPasswordPolicyData_CombineWith_Joins_Policy_Options()
+    {
+        var mpd1 = new MasterPasswordPolicyData { MinLength = 20, RequireLower = false, RequireSpecial = false };
+        var mpd2 = new MasterPasswordPolicyData { RequireLower = true };
+        var mpd3 = new MasterPasswordPolicyData { RequireSpecial = true };
+
+        mpd1.CombineWith(mpd2);
+        mpd1.CombineWith(mpd3);
+
+        Assert.Equal(20, mpd1.MinLength);
+        Assert.True(mpd1.RequireLower);
+        Assert.True(mpd1.RequireSpecial);
+    }
```

Comment:
The SUT here should still be the `MasterPasswordPolicyRequirementFactory`. i.e. call the factory `Create` method with these 3 policies.